packages/genesys-spark-components/src/components/stable/gux-datepicker/gux-datepicker.tsx

```diff
@@ -72,6 +73,7 @@ export class GuxDatepicker {
   startInputId: string = randomHTMLId('gux-datepicker');
   endInputId: string = randomHTMLId('gux-datepicker');
   private cleanupUpdatePosition: ReturnType<typeof autoUpdate>;
+  private focusinFromClick: boolean = false;
```

Comment:
@jason-evans-genesys can you remove all the focus in from click code you added. I was talking to the UX team and they are going to have another look at how we approach this issue.

Reply:
This is done.